contracts-periphery/Makefile

```diff
@@ -0,0 +1,7 @@
+.PHONY: test
+
+build :; forge build
+clean :; forge clean
+install :; forge install
+snapshot :; forge snapshot --match-path test/Gas.t.sol
+test :; forge test
```

Comment:
Running `make test` still results in the gas snapshot "tests" running even though they're not testing anything. To exclude them, perhaps we rename `Gas.t.sol` to `Gas.sol` and then your commands could be:

```
forge snapshot --match-path test/Gas.sol
forge test --match-path test/*.t.sol
```

Reply:
I don't like calling the file `Gas.sol`, because we always use `.t.sol` to represent test contracts and that breaks the convention unnecessarily. Personally I'd suggest either:
1. Undoing this change, because the tests are fast and there's no harm in running them anyway, plus they cover a lot of scenarios we don't test otherwise, OR
2. Rename the file back to `Gas.t.sol` and just change the commands to:

```make
snapshot :; forge snapshot --match-path test/Gas.t.sol
test :; forge test --no-match-path test/Gas.t.sol
```

Reply:
Right... the precedent is `gas.sol` https://github.com/endaoment/endaoment-contracts-v2/blob/1deb992e9778e254f0896086119d691074a6e859/Makefile#L20

Reply:
Also related: A convention I like for foundry script files is `MyScript.s.sol`. In other words, the suffix indicates how you're supposed to execute that file.

Reply:
Renamed to `UmbraBatchSend.gas.t.sol` in https://github.com/ScopeLift/umbra-protocol/pull/338/commits/e478c2700cf0868343209496d59ba03689220ae4

Reply:
It still feels odd to me to run these during the normal test flow but not strongly enough to block on. Good to merge IMO.

Reply:
We can remove them in the future if it starts affecting readability. Let's 🛳️ it.